server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -3386,6 +3386,7 @@ public UsageRecordResponse createUsageResponse(Usage usageRecord, Map<String, Se
                 resourceType = ResourceTag.ResourceObjectType.UserVm;
                 usageRecResponse.setUsageId(vm.getUuid());
                 resourceId = vm.getId();
+                usageRecResponse.setOsTypeId(vm.getGuestOSId());
```

Comment:
@ggoodrich-ipp we just caught that this sets the DB internal ID and not UUID, should we fix that in 4.15.1?

Reply:
I guess I must've realized this, as I'm storing this off as an int in a local billing database. Is it an easy fix to change this to get the UUID? It appears where I'm using it, I fetch this int value, and then use it to go get the UUID, so it seems the UUID is ultimately what I need. Maybe I used the int as that was already in the model? It has been far enough back that I don't recall the specifics.